testEMagLs.m

```diff
@@ -9,40 +9,222 @@
 % This software is licensed under a Non-Commercial Software License 
 % (see https://github.com/thomasdeppisch/eMagLS/blob/master/LICENSE for full details).
 %
-% Thomas Deppisch, 2021
+% Hannes Helmholz, 2023
 
-clear all
-close all
+clear; clc; close all;
 
-addpath(genpath('dependencies/'))
-[smaRecording, ~] = audioread('resources/Acappella_Eigenmike_Raw_32ch_short.wav');
+addpath(genpath('dependencies/'));
 
-%% get filters for the LS, MagLS, eMagLS and eMagLS2 renderers
-shOrder = 4;
+%% configuration
 filterLen = 512;
+applyDiffusenessConst = false; % or e.g. true, for MagLS, eMagLS and eMagLS2
+
+shFunction   = @getSH; % from Spherical-Harmonic-Transform toolbox
+shDefinition = 'real'; % or e.g. 'complex'
+
+% % An alternative version which uses a different SH basis convention and implementation
+% shFunction   = @getSH_AmbiEnc; % from Ambisonic Encoding toolbox
+% shDefinition = 'real'; % or e.g. 'complex'
+
+[hrirFile, hrirUrl] = deal('resources/HRIR_L2702.mat', ...
+    'https://zenodo.org/record/3928297/files/HRIR_L2702.mat');
+
+[smaRecordingLength, smaRecordingFile, smaRecordingUrl] = deal( ...
+    13.5, ... % in seconds
+    'resources/Acappella_Eigenmike_Raw_32ch.wav', ...
+    'https://zenodo.org/record/3477602/files/09%203D-MARCo%20Samples_Acappella.zip');
```

Comment:
This unfortunately downloads a lot of data and takes forever, maybe rather go back to old version with small file directly available in repository?

Reply:
For now, I added a comment to the command line that the download might take a while on the first run.